### ISSUE TITLE
Canceling slow query

### ISSUE THREAD

**Issue by karlb** [amjith/pgcli#77]
When using psql, I can cancel a slow query (e.g. when I accidentally forgot a LIMIT clause) by pressing ctrl-c. This would be useful for pgcli, too.

**Comment by amjith** [comment]
Yes. That would be useful. I'll look into this. 

Thank you!

**Comment by amjith** [comment]
Note to dev:

Check out how to how to hook up [on_abort](https://github.com/jonathanslenders/python-prompt-toolkit/blob/master/prompt_toolkit/__init__.py#L233) to cancel a query. 

[Psycopg2.cancel](http://initd.org/psycopg/docs/connection.html#connection.cancel)

**Comment by amjith** [comment]
This is now available via the pr #92 by @macobo. 

It will be available in the next release. 

If you're itching to try it out you can install from master using the following command: 

```
pip install -e git+git@github.com:amjith/pgcli.git#egg=pgcli
```

**Comment by karlb** [comment]
That was quick! Thanks!

**Comment by amjith** [comment]
This is now released in 0.11.0. 

https://pypi.python.org/pypi?:action=display&name=pgcli&version=0.11.0
